src/modules/AuthModule/components/ResetPass/ResetPass.tsx

```diff
@@ -1,9 +1,278 @@
+                        edge="end"
+                      >
+                        {showPassword ? <VisibilityOff /> : <Visibility />}
+                      </IconButton>
+                    </InputAdornment>
+                  }
+                  label="Password"
+                />
+                {errors.password && (
+                  <Alert sx={{ mt: 1 }} severity="error">
+                    {errors.password.message?.toString()}
+                  </Alert>
+                )}
+
+                <InputLabel
+                  htmlFor="outlined-adornment-confirm-password "
+                  className="my-1"
+                >
+                  Confirm Password
+                </InputLabel>
+                <OutlinedInput
+                  id="outlined-adornment-confirm-password "
+                  fullWidth
+                  placeholder="Please Type Here"
+                  className="bg-[#F5F6F8]"
+                  type={showConfirmPassword ? "text" : "password"}
+                  {...register("confirmPassword", {
+                    required: "confirmPassword is required",
+                    validate: validatePasswordMatch,
+                  })}
+                  endAdornment={
+                    <InputAdornment position="end">
+                      <IconButton
+                        aria-label="toggle password visibility"
+                        onClick={handleClickShowConfirmassword}
+                        onMouseDown={handleMouseDownConfirmPassword}
+                        edge="end"
+                      >
+                        {showConfirmPassword ? (
+                          <VisibilityOff />
+                        ) : (
+                          <Visibility />
+                        )}
+                      </IconButton>
+                    </InputAdornment>
+                  }
+                  label="Confirm Password"
+                />
+                {errors.confirmPassword && (
+                  <Alert sx={{ mt: 1 }} severity="error">
+                    {errors.confirmPassword.message?.toString()}
+                  </Alert>
+                )}
+
+                <Button
+                  type="submit"
+                  fullWidth
+                  variant="contained"
+                  sx={{ mt: 5, mb: 2 }}
+                >
+                  Reset
+                </Button>
+              </Box>
+            </Box>
+          </Grid>
+          <Grid
+            item
+            xs={12}
+            sm={6}
+            md={6}
+            sx={backgroundStyle}
+            className="bg-[url('./assets/images/ForgetPass.jpg')] rounded"
+          >
+            <Box
+              sx={{
+                width: "100%",
+                textAlign: "left",
+                padding: 10,
+                color: "white",
+              }}
+            >
+              <h1 className="font-bold line-clamp-6 md:text-2xl lg:text-4xl">
```

Comment:
We should have at most one h1 element (we should also update it to be typography).